ecf/day.sh

```diff
@@ -11,14 +12,34 @@ set -e
 echo zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz loading modules zzzzzzzzzzzzzzzzzzzzzzz
 #. /usrx/local/Modules/3.2.10/init/bash
 module purge
-module load ips/19.0.5.281 impi/19.0.5
-module load prod_envir/1.1.0
-module load prod_util/1.1.5 
-module load grib_util/1.1.1
-module load bufr_dumplist/2.3.0
-module load dumpjb/5.1.0
-module load NetCDF/4.5.0
-module load lsf/10.1
+module load envvar/1.0
+module load imagemagick/7.0.8-7
+module load prod_envir/2.0.5
+module load prod_util/2.0.9
+#Can be loaded directly -- required for additional modules
+module load intel/19.1.3.304
+module load craype/2.7.8
+module load PrgEnv-intel/8.1.0
+#Additional:
+module load wgrib2/2.0.8
+module load w3nco/2.4.1
```

Comment:
Already at w3emc/2.9.1+ in my fork.  My plan is to first bring this up to what went in to operations (including the outdated references) and then do an update to something closer to my working fork which will ensure that all your suggestions are incorporated. Yes, that's 2 PR instead of 1, so if you'd prefer 1, I can go that way. But I'd prefer to first tag what went in to operations and then do the update .